tests/escrow/programs/escrow/src/lib.rs

```diff
@@ -102,24 +123,23 @@ pub mod escrow {
 #[derive(Accounts)]
 #[instruction(initializer_amount: u64)]
 pub struct InitializeEscrow<'info> {
-    #[account(signer)]
-    pub initializer: AccountInfo<'info>,
+    #[account(mut)]
+    pub initializer: Signer<'info>,
     #[account(
         mut,
         constraint = initializer_deposit_token_account.amount >= initializer_amount
     )]
     pub initializer_deposit_token_account: Account<'info, TokenAccount>,
     pub initializer_receive_token_account: Account<'info, TokenAccount>,
-    #[account(init, payer = initializer, space = 8 + EscrowAccount::LEN)]
-    pub escrow_account: Account<'info, EscrowAccount>,
```

Comment:
I did these change yesterday and launched test in the wrong shell. Reverted locally. Will look into it when I have time. 
And I think it was merged without checking CI.